src/bot/precomileInclude.jl

```diff
@@ -0,0 +1,120 @@
+
+"""
+    precompilePather(packageName::String)
+
+To get the path of precompile_packageName.jl file
+
+Written exclusively for SnoopCompile Github actions.
+# Examples
+```julia
+precompilePath, precompileFolder = precompilePather("MatLang")
+```
+"""
+function precompilePather(packageName::String)
+    return "\"../deps/SnoopCompile/precompile/precompile_$packageName.jl\"",
+    "$(pwd())/deps/SnoopCompile/precompile/"
+end
+
+precompilePather(packageName::Symbol) = precompilePather(string(packageName))
+precompilePather(packageName::Module) = precompilePather(string(packageName))
+
+################################################################
+
+function precompileRegex(precompilePath)
+    # https://stackoverflow.com/questions/3469080/match-whitespace-but-not-newlines
+    # {1,} for any number of spaces
+    c1 = Regex("#[^\\S\\r\\n]{0,}include\\($(precompilePath)\\)")
+    c2 = r"#\s{0,}_precompile_\(\)"
+    a1 = "include($precompilePath)"
+    a2 = "_precompile_()"
+    return c1, c2, a1, a2
+end
+################################################################
+
+"""
+    precompileActivator(packagePath, precompilePath)
```

Comment:
What, exactly, is `packagePath` supposed to be? The name of a file or a directory? Is it always the top-level file? What if the `_precompile_()` isn't in that file but some other file?

Reply:
https://github.com/timholy/SnoopCompile.jl/pull/49#discussion_r362505975 
Should I change the variable name? 

It is the same as `pathof`

Reply:
In this case you can just document it. If it's always `pathof(MyPkg)` then is it feasible to pass `MyPkg` as an argument and call `pathof` here? That would ensure you always get the right answer.

Reply:
We should use something like
```julia
packagePath = Base.read(`cmd /c julia -e 'import MatLang; print(pathof(MatLang))'`, String)
```
if we want to calculate pathof automatically.

Anyways these functions are internal to the bot and are not supposed to be used directly.